docs/usage/configuration-options.md

```diff
@@ -2445,6 +2445,10 @@ Configure this to `true` if you wish to receive one PR for every separate major
 e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
 If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.
 
+## separateMultipleMinor
```

Comment:
We have a section in the FAQ that explains how to "Separate patch releases from minor releases".

The new feature in this PR seems related to this concept as well?
Do we want a new section in the FAQ that links to the new `separateMultipleMinor` config documentation?

https://docs.renovatebot.com/faq/#separate-patch-releases-from-minor-releases